### PR TITLE
chore: Enable Android/RN/iOS code snippets for feature flags

### PR DESCRIPTION
**Pull request by marandaneto**
## Problem

Clients and code snippets were missing for feature flags - android/rn/ios

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/feature-flags/FeatureFlagCodeOptions.tsx [hunk 1]
```diff
@@ -118,7 +118,15 @@ export const OPTIONS: InstructionOption[] = [
...
 
-export const PAYLOAD_LIBRARIES: string[] = [SDKKey.JS_WEB, SDKKey.NODE_JS, SDKKey.PYTHON, SDKKey.RUBY, SDKKey.REACT]
+export const PAYLOAD_LIBRARIES: string[] = [
+    SDKKey.JS_WEB,
+    SDKKey.NODE_JS,
+    SDKKey.PYTHON,
+    SDKKey.RUBY,
+    SDKKey.REACT,
+    SDKKey.ANDROID,
+    SDKKey.REACT_NATIVE,
```
Comment (neilkakkar) [inline frontend/src/scenes/feature-flags/FeatureFlagCodeOptions.tsx:128]:
```suggestion
    SDKKey.REACT_NATIVE,
    SDKKey.IOS,
```

**Review by neilkakkar — APPROVED** [review]

**Comment by marandaneto** [comment]
@neilkakkar fixed the snippets.

**Comment by neilkakkar** [comment]
Looks good 👍

**Comment by neilkakkar** [comment]
how is this changing the experiment banner 🤔

**Comment by marandaneto** [comment]
> how is this changing the experiment banner 🤔

no idea, but this is all CI.
Probably the experiments use the code snippets somewhere.

**Pull request merged**